tools/tcat_ble_client/GENERATING_CERTIFICATES.md

```diff
@@ -0,0 +1,93 @@
+# BBTC X.509 certificates generation
+
+---
+
+# Overview
+
+TCAT uses X.509 Certificate Extensions to provide permissions with certificates.
+
+# Extensions
+
+Extensions were introduced in version 3 of the X.509 standard for certificates. They allow certificates to be customised to applications by supporting the addition of arbitrary fields in the certificate. Each extension, identified by its OID (Object Identifier), is marked as "Critical" or "Non-Critical", and includes the extension-specific data.
+
+# Certificates generation
+
+Thread uses Elliptic Curve Cryptography (ECC), so we use `ecparam` `openssl` argument to generate the keys.
```

Comment:
```suggestion
Thread uses Elliptic Curve Cryptography (ECC), so we use the `ecparam` `openssl` argument to generate the keys.
```

Reply:
@pjasinski990 this one is still open.  Good to go after this one.

Reply:
fixed.